pkgs/tools/system/disk-filltest/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "disk-filltest";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "bingmann";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1vcb43hdln7xlklz1n0fsfp5x1j9pn829wbad4b110hrc7nwrnvm";
+  };
+
+  preBuild = ''
+    substituteInPlace Makefile --replace 'prefix = /usr/local' 'prefix = $(out)'
+  '';
+
+  postInstall = ''
+    install -D -m0644 -t $out/share/doc COPYING README
+    mkdir -p $out/share/man; mv $out/man1 $out/share/man
+  '';
+
+  meta = with stdenv.lib; {
```

Comment:
Also, please consider specifying `meta.platform`, otherwise the package will not be built on Hydra (see [manual ch. 7.1](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes)).